### PR TITLE
Fixed crash on empty class attribute

### LINKED ISSUE #16: Crashes under default config when given '<div class>'
```js
insane('<div class>');
```

You'll get a`TypeError: Cannot read property 'split' of undefined` in `sanitizer.js` from:

```js
if (lkey === 'class' && attrsOk.indexOf(lkey) === -1) {
  value = value.split(' ').filter(isValidClass).join(' ').trim();
  valid = value.length;
} else {
```

This is because `value` is `undefined`. I'd suggest the two lines be changed to:

```js
if (value) {
  value = value.split(' ').filter(isValidClass).join(' ').trim()
}
valid = !value || value.length;
```

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
Fixes #16

### CODE REVIEW

File: test/insane.js [hunk 1]
```diff
@@ -61,6 +61,11 @@ test('only returns tags in the whitelist even if disallowed tag is nested', func
   t.end();
 });
 
+test('allows blank attribute', function (t) {
```
Comment (bevacqua) [inline test/insane.js]:
```suggestion
test('ignores blank attribute', function (t) {
```

Comment (bevacqua) [reply test/insane.js]: What about valid HTML5 like `<input disabled>`?

Comment (JoshuaKGoldberg) [reply test/insane.js]: Great point. I added some test cases to show that these are still allowed based on the `allowedAttributes` setting.

This PR's added logic only adds to the `if (lkey === 'class' && attrsOk.indexOf(lkey) === -1) {` case, so I'm not worried about impacting other attributes.

**Comment by JoshuaKGoldberg** [comment]
Closing as this has aged away and I no longer work at a company that uses this library. If you the reader would like this code, feel free to take it and send your own PR. ❤️

**Pull request closed without merging**